winui/Kanban/cards.md

```diff
@@ -0,0 +1,161 @@
+
+* `Title` - Gets or sets the string value used to display the header of a card in the KanbanModel.
+* `Description` - Gets or sets the string value used to display the description of a card in the KanbanModel.
+* `Category` - Gets or sets the category associated with the card in the KanbanModel.
+* `Assignee` - Gets or sets the assignee associated with the card in the KanbanModel.
+* `Image` - Specifies the image source representing the card's assignee or related entity.
+* `IndicatorColorKey` - Provides unique key with each Kanban card to dynamically assign indicator colors. It is often used in scenarios where cards are categorized or prioritized by color, based on specific key values.
+* `Id` - Used to uniquely identify each Kanban card, allowing for efficient management, tracking, and updates of individual cards within Kanban control.
+* `Tags` - Used to assign multiple tags to a kanban card, which can be used for categorization, filtering, or additional metadata.
+
+{% highlight C# %}
+
+KanbanModel taskDetail = new KanbanModel();
+taskDetail.Title = "UWP Issue";
+taskDetail.Id = "651";
+taskDetail.Description = "Crosshair label template not visible in UWP";
+taskDetail.Category = "Open";
+taskDetail.IndicatorColorKey = "High";
+taskDetail.Tags = new List<string>() { "Bug Fixing" };
+taskDetail.Image = new Image
+{
+    Source = new BitmapImage(new Uri("ms-appx:///Assets/Kanban/People_Circle1.png"))
+};
+
+{% endhighlight %}
+
+Following code snippet is used to define the colors for each key.
+
+{% tabs %}
+{% highlight XAML hl_lines="6 7 8 9 10" %}
+
+<kanban:SfKanban x:Name="kanban"
+                 ItemsSource="{Binding TaskDetails}">
+    <kanban:SfKanban.DataContext>
+        <local:ViewModel/>
+    </kanban:SfKanban.DataContext>
+    <kanban:SfKanban.IndicatorColorPalette>
+        <kanban:KanbanColorMapping Key="Low" Color="Blue"/>
+        <kanban:KanbanColorMapping Key="Normal" Color="Green"/>
+        <kanban:KanbanColorMapping Key="High" Color="Red"/>
+    </kanban:SfKanban.IndicatorColorPalette>
+</kanban:SfKanban>
+
+{% endhighlight %}
+
+{% highlight C# hl_lines="5" %}
+
+List<KanbanColorMapping> indicatorColorPalette = new List<KanbanColorMapping>();
+indicatorColorPalette.Add(new KanbanColorMapping() { Key="Low", Color = Colors.Blue });
+indicatorColorPalette.Add(new KanbanColorMapping() { Key= "Normal", Color = Colors.Green });
+indicatorColorPalette.Add(new KanbanColorMapping() { Key= "High", Color = Colors.Red });
+this.kanban.IndicatorColorPalette = indicatorColorPalette;
+
+{% endhighlight %}
+{% endtabs %}
+
+## Customizing kanban cards
+
+### Customize kanban cards using template
+
+The appearance of Kanban cards can be fully customized using the `CardTemplate` property of the `SfKanban` control. By setting this property, you can define a custom design for the entire card.
+
+The following code snippet demonstrates how to use the `CardTemplate` property to apply a custom template to Kanban cards:
```

Comment:
The following code snippet demonstrates how to use the `CardTemplate` property to apply a custom template to kanban cards: